webhook/psbinding/psbinding.go

```diff
@@ -197,56 +193,48 @@ func (ac *Reconciler) Admit(ctx context.Context, request *admissionv1.AdmissionR
 		return webhook.MakeErrorStatus("unable to decode object: %v", err)
 	}
 
-	// Look up the Bindable for this resource.
-	fb := func() Bindable {
-		ac.lock.RLock()
-		defer ac.lock.RUnlock()
-
-		// Always try to find an exact match first.
-		if sb, ok := ac.exact.Get(exactKey{
-			Group:     request.Kind.Group,
-			Kind:      request.Kind.Kind,
-			Namespace: request.Namespace,
-			Name:      orig.Name,
-		}); ok {
-			return sb
-		}
-
-		// Next look for inexact matches.
-		if sb, ok := ac.inexact.Get(inexactKey{
-			Group:     request.Kind.Group,
-			Kind:      request.Kind.Kind,
-			Namespace: request.Namespace,
-		}, labels.Set(orig.Labels)); ok {
-			return sb
-		}
-		return nil
-	}()
-	if fb == nil {
+	// Look up the Bindables for this resource.
+	fbs := ac.index.lookUp(exactKey{
+		Group:     request.Kind.Group,
+		Kind:      request.Kind.Kind,
+		Namespace: request.Namespace,
+		Name:      orig.Name,
+	},
+		labels.Set(orig.Labels))
```

Comment:
this formatting is a bit odd

Reply:
Agreed. Note that the same oddity occurred in the deleted code just above. Thanks `go fmt`.

Reply:
I'd join with the previous line to avoid this.

Reply:
Fixed two occurrence. Looks a little better now.

Reply:
Fixed.

Reply:
Fixed.